R/utils_general_str_formatting.R

```diff
@@ -318,3 +318,46 @@ is_adjacent_separate <- function(group_1,
 
   return(TRUE)
 }
+
+str_catalog <- function(item_vector,
```

Comment:
where is this function used?

Reply:
It's used in the `validate_style_in()` function (and I'm thinking of using it elsewhere).